cli/cli_cheats_sheet.tex

```diff
@@ -109,17 +107,22 @@
 \\
 %
 \begin{tabularx}{\linewidth}{lX}
+\smallhspace \cliverb{convert}      & Write out a new bag with different settings.\\
 \smallhspace \cliverb{info}      & Output information of a bag.\\
+\smallhspace \cliverb{list}      & Output list of available plugins.\\
 \smallhspace \cliverb{play}      & Play a bag.\\
-\smallhspace \cliverb{record}    & Record a bag.
+\smallhspace \cliverb{record}    & Record a bag.\\
+\smallhspace \cliverb{reindex}    & Reconstruct metadata file for a bag.\\
```

Comment:
```suggestion
\smallhspace \cliverb{reindex}    & Reconstruct metadata file for a bag.
```

Reply:
Fixed in https://github.com/ubuntu-robotics/ros2_cheats_sheet/pull/6/commits/2cb22ee86c7d261c14f6a32456d84f942ecac103